source/stylesheets/_core.scss

```diff
@@ -94,6 +94,19 @@ body {
   width: 165%;
 }
 
+@import url("//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css");
```

Comment:
recommend against this if this is treated as a css import, as the result is that this will be loaded sequentially, and block rendering on screen

Reply:
yeah, this is being converted to a css `import`.

You can either use a `link` tag in the main layout:
```html
<link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
```

or use `font-awesome-sass`: https://fontawesome.com/v4.7.0/get-started/#ruby-gem-sass